### DISCUSSION TITLE
Go version support?

### DISCUSSION THREAD

**Discussion by capnspacehook** [florianl/go-nfqueue#18] [Ideas]
I am working on a patch that would really benefit from `errors.Is`, but I am unsure whether I can use it or not. The `go.mod` file is at Go version 1.13, but you still support Go 1.12 and earlier. Is that a mistake or intentional, and would you be willing to drop support for Go 1.12 and earlier for better handling of errors? If so I could send a PR.

**Comment by florianl** [comment]
With commit 3cf809d1d085a075c68d9fdcfe9c66a8e5462459 I have updated the minimal required Go version to 1.14 and subsequently removed legacy code.
Once PR https://github.com/florianl/go-nfqueue/pull/17 is merged, I plan to tag a new version.

Hope this resolves concerns :)

**Reply by capnspacehook** [reply]
Awesome, yes that's perfect!
